superset-frontend/src/features/home/Menu.tsx

```diff
@@ -154,6 +154,29 @@ const globalStyles = (theme: SupersetTheme) => css`
       margin-left: ${theme.gridUnit * 1.75}px;
     }
   }
+  .ant-menu-item-selected {
+    background-color: transparent !important;
```

Comment:
Is it possible to avoid `!important` here? Maybe using a more specific class?

Reply:
@michael-s-molina I just removed all these redundant `!important`